### ISSUE TITLE
Scrollbar is getting too much space depeing on OS and Browser

### ISSUE THREAD

**Issue by ruizmarc** [MurhafSousli/ngx-scrollbar#147]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?

It depends on OS. For example, native scrollbar in MacOS width is 0px, native scrollbar in KDE Neon is 15px. But ngx-scrollbar is assuming 18px for any browser and OS. 

https://github.com/MurhafSousli/ngx-scrollbar/blob/fe794297767b975d884e64fe39d3f91b50553f16/projects/ngx-scrollbar/src/scrollbar/ng-scrollbar.ts#L22-L25

### Versions

Angular 7.2.1
Ngx-scrollbar 4.2

### Repro steps

As it depends on user environment (OS and Browser). I'm attaching three screenshots of the exact same view in three different Browsers and OS. 

MacOS - Safari 12.1.1
![macos-safari-12 1 1](https://user-images.githubusercontent.com/5717082/58183004-e171dc00-7cae-11e9-9f52-4d159809cf15.png)

MacOS - Chrome 74
![macos-chrome74](https://user-images.githubusercontent.com/5717082/58183012-e33b9f80-7cae-11e9-9b23-04f815d33fc8.png)

KDE Neon - Chrome 74
![kde-neon-chrome74](https://user-images.githubusercontent.com/5717082/58183026-e6cf2680-7cae-11e9-83e7-572ce1bc918f.png)

### Desired functionality

I'm attaching an screenshot to show how we expect to see it.
![expected](https://user-images.githubusercontent.com/5717082/58183243-488f9080-7caf-11e9-93b8-6b8b2933b16f.png)

It would be great if instead of setting the width of the scrollbar, it can be calculated.

### Mention any other details that might be useful

I'm attaching an example of scrollbar width calculation, just in case it may help. 

http://jsfiddle.net/UU9kg/17/

Thanks in advance for your work :D

**Comment by Nekronik** [comment]
It has also been driving me crazy on a full-screen google map page.
![Screenshot 2019-05-22 at 16 45 15](https://user-images.githubusercontent.com/7140454/58184340-34e52980-7cb1-11e9-88e4-39d477a2dd5f.png)

**Comment by cluetjen** [comment]
Scrollbar width calculation could fix #127 too

**Comment by MurhafSousli** [comment]
@cluetjen No that won't fix that issue, scrollbar size value won't change if user zoomed in the page
